test/q-transformer.test.js

```diff
@@ -1129,8 +1120,8 @@ describe('qTransformer', () => {
                             text: "You could be prosecuted or get less compensation if you give false or misleading information."
                         }) }}`
                 };
-                expect(removeIndentation(result.transformation)).toEqual(
-                    removeIndentation(expected.transformation)
+                expect(removeIndentation(result.content)).toEqual(
```

Comment:
Some the tests aren't fully testing the response. Can you update them to include the title too. I've notice ones that would fail as they are e.g.

pageTitle: 'Error: Email address - claim criminal injuries compensation - GOV.UK',

Should be something like:

pageTitle: '{{ ... }}Email address',